packages/js/e2e-environment/config/jest-object.config.js

```diff
@@ -3,7 +3,7 @@
  */
 const { E2E_RETRY_TIMES } = process.env;
 
-const setupJestRetries = ( retries = 2 ) => {
+const setupJestRetries = ( retries = 0 ) => {
```

Comment:
We should make the `jest.retryTimes()` call conditional on `retryTimes > 0`. That way, when the function is called locally and in the package, it doesn't matter which order they are called in. By default, in the package call `retries` will be `0`. If it's called after a local call that enables them, the default will turn them back off.

Reply:
That's great forward thinking. Since `retryTimes` is set right above using this command `const retryTimes = E2E_RETRY_TIMES ? E2E_RETRY_TIMES : retries;`, I'll just use `retryTimes > 0` as the condition.

Reply:
> I'll just use retryTimes > 0 as the condition.

We want to be able to disable retries with `E2E_RETRY_TIMES=0` when they are enabled in the project jest setup.